src/Stratis.Bitcoin.Features.Wallet.Tests/WalletTransactionHandlerTest.cs

```diff
@@ -148,7 +148,13 @@ public void BuildTransactionFeeTooLowThrowsWalletException()
         [Fact]
         public void BuildTransactionNoChangeAdressesLeftCreatesNewChangeAddress()
         {
-            (Wallet wallet, (ExtKey ExtKey, string ExtPubKey) accountKeys, (PubKey PubKey, BitcoinPubKeyAddress Address) destinationKeys, TransactionData addressTransaction, WalletTransactionHandler walletTransactionHandler, WalletAccountReference walletReference) = this.SetupWallet();
+            WalletTransactionHandlerTestContext testContext = SetupWallet();
+            Wallet wallet = testContext.Wallet;
```

Comment:
@codingupastorm why create variables in each test when you can just pass the `WalletTransactionHandlerTestContext` instance to `CreateContext`?

Reply:
Or whatever is required...

Reply:
Ah, in the future we may have other params to add to CreateContext, so it doesn't really make sense. 

I'm thinking this after working on the TransactionBuildOptions coming in a new PR (hopefully tomorrow!)

Reply:
What I mean is, can this test be written like this?

```
    WalletTransactionHandlerTestContext testContext = SetupWallet();
    Wallet wallet = testContext.Wallet;
    (ExtKey ExtKey, string ExtPubKey) accountKeys = testContext.AccountKeys;
    WalletTransactionHandler walletTransactionHandler = testContext.WalletTransactionHandler;
    TransactionData addressTransaction = testContext.AddressTransaction;

    TransactionBuildContext context = CreateContext(this.Network, testContext.WalletReference, "password", testContext. DestinationKeys.PubKey.ScriptPubKey, new Money(7500), FeeType.Low, 0);
    ...

```

etcetc

Reply:
Sure, I can do that. I was very lazy

Reply:
Done :)